### ISSUE TITLE
[NPC/Event] Stormwind Stockades RP event not working.

### ISSUE THREAD

**Issue by rzeg** [Crestfall/JadefireTracker#93]
Seems the RP event is severely messed up as several NPC's keep resetting in and out of combat.

![d](http://puu.sh/ijzQx/9f0bf9bf3f.jpg)

**Comment by AsuraDev** [comment]
This is correct. It was unfinished. I'll remember to finish it one of these days :P
